### ISSUE TITLE
Polygon doesn't support an icons property

### ISSUE THREAD

**Issue by krewmarco** [angular-ui/angular-google-maps#1459]
The documentation at 

http://angular-ui.github.io/angular-google-maps/#!/api/polygon

shows an 'icons' property for polygons, but the 'icons' property doesn't appear supported by the Google Maps API:

https://developers.google.com/maps/documentation/javascript/3.exp/reference#PolygonOptions

I discovered this after being unable to get the 'icons' property to display anything on a polygon.

**Comment by nmccready** [comment]
Its a typo in the docs then. It's not in the code either.

https://github.com/angular-ui/angular-google-maps/blob/master/src/coffee/directives/api/i-polygon.coffee#L7-L20

**Comment by nmccready** [comment]
Feel free to remove it and PR to the website branch.

**Comment by nmccready** [comment]
Wait let me get my head on straight.. its in the code; I'll have to look into it.

**Comment by nmccready** [comment]
You are correct; this is just a copy and paste error as the functionality of polyline and polygon is very similar.

**Comment by nmccready** [comment]
Feel free to fix it in the code base. I'll get to it eventually.

**Comment by nmccready** [comment]
@pcbulai can u update the website for that? I can do the code.

**Comment by pcbulai** [comment]
@nmccready sure thing

**Comment by markmssd** [comment]
Another copy/paste error (just a guess), is the fact that Polygon does not make any distinction between ```fillColor``` and ```strokeColor```. It only has a ```color``` property like the Polyline directive.

https://developers.google.com/maps/documentation/javascript/reference?csw=1#PolygonOptions

I tried to add it myself but no luck. If you could point me out in the right direction, I would be happy to PR it once done, thanks!
